onnxruntime/test/onnx/main.cc

```diff
@@ -116,7 +116,7 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
           enable_cpu_mem_arena = false;
           break;
         case 'v':
-          logging_level = ORT_LOGGING_LEVEL_INFO;
+          logging_level = ORT_LOGGING_LEVEL_VERBOSE;
```

Comment:
By default, it's ERROR.
'-v' become INFO
'-vv' should be VERBOSE.

Reply:
Ok, will fix.